src/index.js

```diff
@@ -1,10 +1,12 @@
 import useAsyncOps from './useAsyncOps'
 import { register } from './operations'
 import { enable as enableMock, disable as disableMock } from './mock'
+import Provider from './Provider'
 
 export {
   useAsyncOps,
   register,
   enableMock,
-  disableMock
+  disableMock,
+  Provider
```

Comment:
I think we should rename this to something less generic that fits its function, maybe `AsyncOpsStateProvider`